lib/utils/ssh.ts

```diff
@@ -36,17 +35,41 @@ export async function exec(
 	cmd: string,
 	stdout?: NodeJS.WritableStream,
 ): Promise<void> {
-	const command = `ssh \
-		-t \
-		-p 22222 \
-		-o LogLevel=ERROR \
-		-o StrictHostKeyChecking=no \
-		-o UserKnownHostsFile=/dev/null \
-		root@${deviceIp} \
-		${cmd}`;
+	const { which } = await import('./helpers');
+	const program = await which('ssh');
```

Comment:
This call to `which()` replaces the previous call to `getSubShellCommand()`. It allows us to avoid using a shell (`cmd.exe` on Windows or `/bin/sh` on Unix) to invoke `ssh`, which removes the need to shell-escape arguments, which was the cause of one of the bugs on Windows.